lib/lighthouse/facilities/v1/response.rb

```diff
@@ -21,7 +21,7 @@ def initialize(body, status)
           self.body = body
           self.status = status
           parsed_body = JSON.parse(body)
-          self.data = parsed_body['data']
+          self.data = parsed_body['data'] || []
```

Comment:
If no facilities are returned, the lighthouse api does not return a data key or object. This handles that scenario by setting it to an empty array.

Reply:
Yep! It's only an issue with the pagination params with a page that is outside of the total results (ie give me page 10 with 10 per page of a total result set of 5).